components/automate-ui/src/app/modules/user/user.module.ts

```diff
@@ -9,6 +9,7 @@ import { UserManagementComponent } from './user-management/user-management.compo
 import { UserTableComponent } from './user-table/user-table.component';
 import { UserDetailsNonAdminResolve } from './user-details/user-details.resolver';
 
+import { TelemetryCheckboxModule } from 'app/page-components/telemetry-checkbox/telemetry-checkbox.module';
```

Comment:
Import component here

Reply:
Same component not allowed to add in multiple modules, so I created separate module and import it here.